src/utils/RouteUtils.js

```diff
@@ -56,13 +56,14 @@ async function getRoutes(
   isArriveBy: boolean,
 ): Promise<Array<Object>> {
   const routeResponses = await GhopperUtils.fetchRoutes(end, start, departureTimeQuery, isArriveBy);
+  const { busRoute } = routeResponses;
```

Comment:
can this be combined into one like megan's comment in your sectioned routes pr

Reply:
good catch. I changed walkingRoute to a const in that PR so once I merge that, I'll combine it into one line here

Reply:
too speedy